tools/case_id_splitter.rb

```diff
@@ -95,6 +95,22 @@ def run
         end
       end
 
+      command :"add-tags" do |c|
+        c.syntax = "#{$PROGRAM_NAME} add-tag [tag] [test_case]"
```

Comment:
Here is using `add-tag`, the others are using `add-tags`

Reply:
Thanks, corrected.